### ISSUE TITLE
Codelense on light theme is less than 4.5:1

### ISSUE THREAD

**Issue by Blessymanna** [microsoft/vscode#137429]
[Check out Accessibility Insights!](https://accessibilityinsights.io/ )- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11yMAS;#A11y_VSCode;#A11ySev3;#A11yTCS;#A11yWCAG;#DesktopApp;#Visual Studio Code Client;#ColorContrast;#WCAG1.4.3;#DesktopApp;#Win10;#VisualStudioCode-Win32-Apr2021;

### Environment details:
Application Name: Azure Stream Analytics Tools for Visual Studio Code
Windows Version: Win10
Visual studio code version : -1.62.2(User Setup)

### Repro Steps:

1. Launch Visual Studio code application with installed Azure Stream Analytics Tools extension .
2. Tab to "Explorer" section in activity bar.
3. Press control +shift +P and edit bar will appear.
4. Type "ASA:Create new project" and press enter to create project.
5. New project will get created.
6. Tab to Input item in Project Tree Section
7. Right click on input and navigate to "ASA : Add Input" using arrow key and press enter.
8. Select IoT hub from available option using enter key.
9. Select "select from azure subscriptions" from available option using enter key.
10. Type any name for input file and press enter to create Input json file.
11. Tab to "Select Subscription" button.
12. Start AI4D tool and check the color contrast ratio of select subscription text.

### Actual Result:
The color contrast ratio of the text "Select Subscription" button available in input json file is less than 4.5:1.

Note: - This issue is also observed with "Select IoT Hub", "Select Shared Access Policy name" text

### Expected Result:
The color contrast ratio of the text "Select Subscription" button available in input json file should be greater than or equal to 4.5:1.

### User Impact:
The low vision users may not able to see the "Select Subscription" text available in the input json file.

**Comment by Blessymanna** [comment]
The bugs logged on the same issue has been closed and the issue hasn't fixed yet. As per the discussion with our SME's, logged this bug and according to their input this needs to be fixed.

**Comment by IllusionMH** [comment]
Other issues are marked as duplicates of #126899 and https://github.com/microsoft/vscode/issues/126899#issuecomment-866041214 has explanation why it's only 3:1 and not 4.5:1

> Since the code lens is designed to be not annoying, I suggest that we go with the minimum required contrast of 3:1.
Not the recommend contrast of 4.5:1
More about required contrast ratios can be found here
